guides/v2.1/install-gde/prereq/prereq_updater.md

```diff
@@ -1,18 +1,163 @@
+
+1.  Copy to the clipboard the Magento 2 Updater repository SSH clone URL.
+    
+    a. In a web browser, go to [the Magento 2 Updater repository][2]{: target="_blank"}.
+    
+    b. On the right side of the page, under the *clone URL* field, click **SSH**.
+    
+    c. Click the **Copy to clipboard** button.
+    
+    The following figure shows an example.
+    
+    ![Clone the Magento 2 Updater repository using SSH]({{ site.baseurl }}/common/images/install_mage2_clone-ssh.png){: width="650px"}
+
+2.  Change to your web server's docroot directory.
+    
+    Typically, for Ubuntu, it's `/var/www` and for CentOS it's `/var/www/html`.
+    
+    Need help locating the docroot? Click [here.]({{ page.baseurl }}/install-gde/basics/basics_docroot.html)
+
+3.  Enter the following command:
+    ```bash
+    git clone git@github.com:magento/magento2-updater.git
+    ```
+
+4.  Wait for the repository to clone on your server.
+    
+    <div class="bs-callout bs-callout-info" id="info" markdown="1">
+    If the following error displays, make sure you [shared your SSH key][1]{: target="_blank"} with GitHub:
+    ```terminal
+    Cloning into 'magento2'...
+    Permission denied (publickey).
+    fatal: The remote end hung up unexpectedly
+    ```
+    </div>
+
+5.  Continue with [Update installation dependencies](#compman-clone-composer).
+
+### Clone with HTTPS   {#compman-clone-https}
+
+To clone the Magento 2 Updater repository using the HTTPS protocol:
+
+1.  Copy to the clipboard the Magento 2 Updater repository HTTPS clone URL.
+    
+    a. In a web browser, go to [the Magento 2 Updater repository][2]{: target="_blank"}.
+    
+    b. On the right side of the page, under the *clone URL* field, click **HTTPS**.
+    
+    c. Click the **Copy to clipboard** button.
+    
+    The following figure shows an example.
+    
+    ![Clone the Magento 2 Updater repository using HTTPS]({{ site.baseurl }}/common/images/install_mage2_clone-https.png)
+
+2.  Change to your web server's docroot directory.
+    
+    Typically, for Ubuntu, it's `/var/www` and for CentOS it's `/var/www/html`.
+
+3.  Enter the following command:
+
+    ```bash
+    git clone https://github.com/magento/magento2-community-edition.git
+    ```
+    
+    If the command fails with an authentication error, try
+    ```bash
+    git clone https://<your GitHub user name>:<your password>@github.com/magento//magento2-community-edition.git
+    ```
+    
+4.  Wait for the repository to clone on your server.
+
+5.  Continue with the next section.
+
+### Update installation dependencies   {#compman-clone-composer}
+
+The updater application has dependencies specified in its `composer.json` file. To update it, you must authentication. The `composer install` commands fails if you do not. To authenticate, [generate authentication keys]({{ page.baseurl }}/install-gde/prereq/connect-auth.html), after which you create an `auth.json` file in the home directory of the Magento file system owner.
```

Comment:
`you must authentication` - looks like a typo.
In general, "authenticate" where? I think this information is outdated, no authentication should be needed.